src/js/actions/AppsActions.js

```diff
@@ -43,6 +43,14 @@ var AppsActions = {
       });
   },
   createApp: function (newAppAttributes) {
+    if (newAppAttributes == null) {
+      AppDispatcher.dispatch({
+        actionType: AppsEvents.CREATE_APP_ERROR,
+        data: {status: 400}
```

Comment:
I wanted to limit the impact of this simple fix, but you're right I shouldn't have to pass a fake error response. Thanks for the patch! 

*BTW: This once more a good example, why we should introduce a service/api layer to handle all the api calls.*